modules/social_features/social_group/social_group.install

```diff
@@ -1080,3 +1080,17 @@ function social_group_update_12003(): string {
   // Output logged messages to related channel of update execution.
   return $updater->logger()->output();
 }
+
+/**
+ * Enable pager limit on the manage members page.
+ */
+function social_group_update_12004(): string {
```

Comment:
@volodymyr-sydor since we move this PR to the milestone 12.1.0 would be also rename hook update number to the `121001`

Reply:
@ribel done